ideas-list.md

```diff
@@ -248,12 +248,70 @@ This project aims to develop an open-source, autonomous AI Integration Engineer,
 - A foundational understanding of how LLMs work.
 - Familiarity with applied generative AI concepts and techniques.
 - Basic understanding of integration concepts.
-- Knowledge of WSO2 Synapse and Ballerina. (While not strictly required, prior experience will be highly beneficial.)
+- Knowledge of WSO2 Synapse (or willingness to learn).
 - Experience with shell scripting.
 - Familiarity with API development and RESTful principles.
 
 ---
 
+### 🚀 8. ** Templating Library for Ballerina **  
+
+**📅 Project Duration:** Long (~350 hours)  
+**🔥 Difficulty Level:** Medium
+**👨‍🏫 Number of people:** 1
+**👨‍🏫 Mentors:** Anjana Supun (anjanas@wso2.com), Chiran Sachintha (chirans@wso2.com)
```

Comment:
Shall we add them to the mentors.md file as well?

Reply:
Added